macOS/DuckDuckGo/Common/Localizables/UserText.swift

```diff
@@ -1290,6 +1290,7 @@ struct UserText {
 
     // Favorites
     static let newTabFavoriteSectionTitle = NSLocalizedString("newTab.favorites.section.title", value: "Favorites", comment: "Title of the Favorites section in the home page")
+    static let newTabOmnibarSectionTitle = NSLocalizedString("newTab.favorites.section.omnibar", value: "Search", comment: "Title of the Search section in the home page")
```

Comment:
In C-S-S 10.6.0, it’s called `Search` instead of `Search & Duck.ai` so I’ve aligned it with the newer name.